### PR TITLE
Use correct colors for info card, add icon, remove unused icon.

### PR DESCRIPTION
**Pull request by hugokallstrom**
Fix path for downloaded file.
Set action bar type for scaffold.
Fix designs.

### CODE REVIEW

**Review by StylianosGakis — CHANGES_REQUESTED** [review]
I think some important notes around theming, especially for the changed colors in the dark color scheme which I think we'd need to not merge in. And continuing from that, some of the usages of those colors I am curious about.

File: app/app/src/main/kotlin/com/hedvig/app/feature/home/ui/HomeFragment.kt [hunk 1]
```diff
@@ -604,33 +623,32 @@ private fun CommonClaimsRenderer(
...
-                    else -> error("Impossible")
-                  }.themedIcon,
-                )
-                .size(with(density) { 24.dp.roundToPx() })
-                .build(),
-              contentDescription = null,
-              imageLoader = imageLoader,
-              modifier = Modifier.size(24.dp),
-            )
-            Spacer(Modifier.height(8.dp))
```
[the remark this answers is not in the document]
Comment (hugokallstrom) [reply app/app/src/main/kotlin/com/hedvig/app/feature/home/ui/HomeFragment.kt:625]: Makes sense, good catch

File: app/core/design-system/src/main/kotlin/com/hedvig/android/core/designsystem/component/textfield/HedvigTextFieldDefaults.kt [hunk 2]
```diff
@@ -95,9 +95,7 @@ object HedvigTextFieldDefaults {
     shape: Shape = HedvigTextFieldDefaults.shape,
   ) {
     Box(
-      Modifier
-        .background(colors.containerColor(value, enabled, isError, interactionSource).value, shape)
-        .borderLine(enabled, isError, interactionSource, colors, shape),
```
[the remark this answers is not in the document]
Comment (hugokallstrom) [reply app/core/design-system/src/main/kotlin/com/hedvig/android/core/designsystem/component/textfield/HedvigTextFieldDefaults.kt:100]: Yeah after syncing with William we decided to remove this. Its not present on web either so maybe its not used at all.

File: app/core/design-system/src/main/kotlin/com/hedvig/android/core/designsystem/material3/HedvigMaterial3ColorScheme.kt [hunk 3]
```diff
@@ -69,10 +69,10 @@ internal fun darkHedvigColorScheme(
   typeContainer = green_800, // green_100,
   onTypeContainer = green_100, // green_800,
 
-  infoElement = blue_400, // blue_600,
+  infoElement = blue_100, // blue_600,
   onInfoElement = greyscale_0, // greyscale_1000,
   infoContainer = blue_800, // blue_200,
-  onInfoContainer = blue_100, // blue_900,
+  onInfoContainer = blue_900, // blue_900,
```
[the remark this answers is not in the document]
Comment (hugokallstrom) [reply app/core/design-system/src/main/kotlin/com/hedvig/android/core/designsystem/material3/HedvigMaterial3ColorScheme.kt:75]: Oh my bad, was meaning to change light mode only.

File: app/core/ui/src/main/kotlin/com/hedvig/android/core/ui/infocard/InfoCard.kt [hunk 4]
```diff
@@ -57,13 +59,14 @@ fun VectorInfoCard(
...
           style = MaterialTheme.typography.bodyMedium,
+          color = MaterialTheme.colorScheme.onInfoElement,
         )
         Spacer(modifier = Modifier.padding(2.dp))
       }
       Text(
         text = text,
         style = MaterialTheme.typography.bodyMedium,
-        color = MaterialTheme.colorScheme.secondary,
+        color = MaterialTheme.colorScheme.onInfoElement,
```
[the remark this answers is not in the document]
Comment (hugokallstrom) [reply app/core/ui/src/main/kotlin/com/hedvig/android/core/ui/infocard/InfoCard.kt:69]: Ah that works great, thanks. VectorInfoCard just means we can pass a ImageVector icon. We have a DrawableInfoCard as well :)

File: app/feature/travel-certificate/src/main/kotlin/com/hedvig/android/feature/travelcertificate/ui/GenerateTravelCertificateInput.kt [hunk 5]
```diff
@@ -181,97 +183,26 @@ fun GenerateTravelCertificateInput(
           Row(
             horizontalArrangement = Arrangement.SpaceBetween,
             verticalAlignment = CenterVertically,
-            modifier = Modifier.fillMaxWidth(),
+            modifier = Modifier
+              .fillMaxWidth()
+              .height(32.dp),
```
[the remark this answers is not in the document]
Comment (hugokallstrom) [reply app/feature/travel-certificate/src/main/kotlin/com/hedvig/android/feature/travelcertificate/ui/GenerateTravelCertificateInput.kt:188]: Good catch!

File: app/core/design-system/src/main/kotlin/com/hedvig/android/core/designsystem/component/textfield/HedvigTextFieldDefaults.kt (hunk 2 above)
[the remark this answers is not in the document]
Comment (StylianosGakis) [reply app/core/design-system/src/main/kotlin/com/hedvig/android/core/designsystem/component/textfield/HedvigTextFieldDefaults.kt:100]: Aha makes sense. Then we can also remove this https://github.com/HedvigInsurance/android/blob/cd2ac58c9925bbaaf837caa2ae01b88eda8c544d/app/core/design-system/src/main/kotlin/com/hedvig/android/core/designsystem/component/textfield/HedvigTextField.kt#L837 and this https://github.com/HedvigInsurance/android/blob/f068ed72af498799bcb245dcedeb923fbd86b4d2/app/core/design-system/src/main/kotlin/com/hedvig/android/core/designsystem/component/textfield/HedvigTextFieldDefaults.kt#L104-L147 and if there's anything else that would be unused after this change.

File: app/core/ui/src/main/kotlin/com/hedvig/android/core/ui/infocard/InfoCard.kt (hunk 4 above)
[the remark this answers is not in the document]
Comment (StylianosGakis) [reply app/core/ui/src/main/kotlin/com/hedvig/android/core/ui/infocard/InfoCard.kt]: Aha of course! I really didn't put 1 and 1 together back then to understand what Vector means there 😅 Thanks for explaining!

**Review by StylianosGakis — APPROVED** [review]
Nice! Thanks for addressing all the comments 🙏

**Pull request merged**